src/test/java/org/opensearch/sdk/TestExtensionsRunner.java

```diff
@@ -100,8 +100,8 @@ public void testHandlePluginsRequest() throws UnknownHostException {
             emptySet(),
             Version.CURRENT
         );
-        PluginRequest pluginRequest = new PluginRequest(sourceNode, null);
-        PluginResponse response = extensionsRunner.handlePluginsRequest(pluginRequest);
+        InitializeExtensionsRequest pluginRequest = new InitializeExtensionsRequest(sourceNode, null);
+        InitializeExtensionsResponse response = extensionsRunner.handlePluginsRequest(pluginRequest);
```

Comment:
Same as Dan suggested. 
Rename everything Plugin to Extension. 
```suggestion
        InitializeExtensionsResponse response = extensionsRunner.handleExtensionInitRequest(extensionInitRequest);
```